core/src/main/scala/cats/package.scala

```diff
@@ -58,4 +58,14 @@ package object cats {
   val Semigroup = algebra.Semigroup
   val Monoid = algebra.Monoid
   val Group = algebra.Group
+
+  implicit val partialOrderInstances: cats.functor.Contravariant[PartialOrder] =
+    new cats.functor.Contravariant[PartialOrder] {
+      def contramap[A, B](fa: PartialOrder[A])(f: B => A): PartialOrder[B] = fa.on(f)
+    }
+
+  implicit val orderInstances: cats.functor.Contravariant[Order] =
+    new cats.functor.Contravariant[Order] {
+      def contramap[A, B](fa: Order[A])(f: B => A): Order[B] = fa.on(f)
+    }
```

Comment:
I'm not 100% sure where instances for types from Algebra belong - happy to move them somewhere a bit less ad-hoc